src/api/app/views/webui/theme/bratwurst/layouts/webui/_navbar.html.haml

```diff
@@ -1,14 +1,14 @@
-.navbar.navbar-default.navbar-fixed-top{:role => "navigation"}
+.navbar.navbar-default.navbar-fixed-top{role: "navigation"}
   / Brand and toggle get grouped for better mobile display
   .container
     .navbar-header
-      %button.navbar-toggle{"data-target" => ".navbar-ex1-collapse", "data-toggle" => "collapse", :type => "button"}
+      %button.navbar-toggle{"data-target" => ".navbar-ex1-collapse", "data-toggle" => "collapse", type: "button"}
```

Comment:
Wouldn't that change you propose be part of another type of fix than "Get rid of 'RuboCop: Use the new Ruby 1.9 hash syntax.' warnings in Haml-lint."?

Can we agree in accept this pull request, and add your change in another one?